sts3x/sts3x_example_usage.c

```diff
@@ -29,15 +29,10 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-/* #include <stdio.h> // printf
- * #include <unistd.h> // sleep
- */
-#include "sts3x.h"
+#include <stdio.h>   // printf
+#include <unistd.h>  // sleep
 
-/**
- * TO USE CONSOLE OUTPUT (PRINTF) AND WAIT (SLEEP) PLEASE ADAPT THEM TO YOUR
- * PLATFORM
- */
```

Comment:
Can you add the following below the printf include:
```
/* TO USE CONSOLE OUTPUT (printf) YOU MAY NEED TO ADAPT THE
 * INCLUDE ABOVE OR DEFINE IT ACCORDING TO YOUR PLATFORM.
 * #define printf(...)
 */
```

Reply:
Done.